src/components/manage/Sidebar/SidebarPopup.jsx

```diff
@@ -2,11 +2,27 @@ import React from 'react';
 import { Portal } from 'react-portal';
 import { CSSTransition } from 'react-transition-group';
 import PropTypes from 'prop-types';
+import { doesNodeContainClick } from 'semantic-ui-react/dist/commonjs/lib';
 
 const DEFAULT_TIMEOUT = 500;
 
-const SidebarPopup = (props, ref) => {
+const SidebarPopup = (props) => {
   const { children, open, overlay } = props;
+
+  const asideElement = React.createRef();
+
+  const handleClickOutside = (e) => {
+    if (asideElement && doesNodeContainClick(asideElement.current, e)) return;
+    props.closeObjectBrowser();
```

Comment:
@sneridagh I think its okay with handling open/close inside sidebarPopup. We should also change the name of closer prop (`props.closeObjectBrowser()`)as the component can be used elsewhere apart from ObjectBrowser.

Reply:
Indeed, that is true. Will look into it.